storage/google_cloud/google_cloud_utils.py

```diff
@@ -116,3 +116,25 @@ def upload_file_to_blob(bucket_credentials_file_path, target_blob_url, f, max_re
         f.seek(0)
         upload_file_to_blob(bucket_credentials_file_path, target_blob_url, f,
                             max_retries - 1, blob_chunk_size / 2)
+
+def list_blobs(target_bucket_url, prefix, bucket_credentials_file_path):
```

Comment:
```suggestion
def list_blobs(bucket_url, prefix, bucket_credentials_file_path):
```
I'm not sure 'target' is helpful here - we used it elsewhere because that was where a file was being uploaded to

Reply:
I agree bucket_url is enough for this.